src/main/java/com/spotinst/sdkjava/model/api/admin/messageCenter/ApiBulkUpdateMessages.java

```diff
@@ -0,0 +1,56 @@
+package com.spotinst.sdkjava.model.api.admin.messageCenter;
+
+import com.fasterxml.jackson.annotation.*;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+import com.spotinst.sdkjava.model.bl.admin.messageCenter.Entities;
```

Comment:
should import api.admin.messageCenter.apiEntities, not this one.

Reply:
It is an unused import removed now